cmd/provisioner-localpv/app/provisioner_blockdevice.go

```diff
@@ -67,8 +68,10 @@ func (p *Provisioner) ProvisionBlockDevice(opts pvController.VolumeOptions, volu
 	// TODO
 	// VolumeMode will always be specified as Filesystem for host path volume,
 	// and the value passed in from the PVC spec will be ignored.
-	fs := v1.PersistentVolumeFilesystem
```

Comment:
Please remove the above TODO as this PR addresses the concern of ignoring the volumeMode.